sources/packages/backend/apps/db-migrations/src/sql/ApplicationOfferingChangeRequests/Create-application-offering-change-requests.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE sims.application_offering_change_requests (
+  id SERIAL PRIMARY KEY,
+  application_id INT NOT NULL REFERENCES sims.applications(id),
+  requested_offering_id INT NOT NULL REFERENCES sims.education_programs_offerings(id),
+  active_offering_id INT NOT NULL REFERENCES sims.education_programs_offerings(id),
+  application_offering_change_request_status sims.application_offering_change_request_status_types NOT NULL,
+  assessed_date TIMESTAMP WITH TIME ZONE,
+  assessed_by INT REFERENCES sims.users(id),
+  student_action_date TIMESTAMP WITH TIME ZONE,
+  reason VARCHAR(500) NOT NULL,
+  assessed_note_id INT REFERENCES sims.notes(id),
+  -- Audit columns
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW (),
+  updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW (),
+  creator INT NULL DEFAULT NULL REFERENCES sims.users(id),
+  modifier INT NULL DEFAULT NULL REFERENCES sims.users(id)
+);
+
+-- ## Comments
+COMMENT ON TABLE sims.application_offering_change_requests IS 'Represents the list of application specific offering change requests, which is requested by institution.';
+
+COMMENT ON COLUMN sims.application_offering_change_requests.id IS 'Auto-generated sequential primary key column.';
+
+COMMENT ON COLUMN sims.application_offering_change_requests.application_id IS 'Foreign key reference to the applications table. Application on which the application specific offering change was requested.';
+
+COMMENT ON COLUMN sims.application_offering_change_requests.requested_offering_id IS 'Foreign key reference to the education programs offerings table. The new or existing offering assigned by the institution for the application specific offering change.';
+
+COMMENT ON COLUMN sims.application_offering_change_requests.active_offering_id IS 'Foreign key reference to the education programs offerings table. The actual offering of the application that was requested for the application specific offering change.';
```

Comment:
`The active offering of the application when application specific offering change was requested.` ?

Reply:
this is what we discussed just now @dheepak-aot.